vpc/service/delete_unused_static_enis.go

```diff
@@ -0,0 +1,94 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/Netflix/titus-executor/logger"
+	"github.com/Netflix/titus-executor/vpc/service/ec2wrapper"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/pkg/errors"
+)
+
+var staticENILastReconciled = map[string]time.Time{}
+
+func (vpcService *vpcService) deleteUnusedStaticENILoop(ctx context.Context, protoItem keyedItem) error {
```

Comment:
We can do this by hand after we've verified everything is okay.

Reply:
ok, fixed